src/Modal.php

```diff
@@ -147,6 +147,19 @@ public function setOptions($options)
         return $this;
     }
 
+    /**
+     * Whether any change in modal DOM should automatically refresh cached positions.
+     * Allow modal window to add scrolling when adding content dynamically after modal creation.
+     *
+     * @return $this
+     */
+    public function observeChanges()
+    {
+        $this->setOptions(['observeChanges' => true]);
```

Comment:
Hello @lubber-de, when is `observeChanges: false` helpful? Are there any negative performance/functional implications when set `observeChanges: true` by default?

the only related discussion I found: https://github.com/fomantic/Fomantic-UI/issues/564

Reply:
observerChanges is also helpful if your code possibly changes input fields (adding/removing) to make sure focus trap will remain working. https://github.com/fomantic/Fomantic-UI/pull/2036
If setting this to true by default might has a  signifant performanc impact depends on how complex your code/Dom is.
https://stackoverflow.com/questions/31659567/performance-of-mutationobserver-to-detect-nodes-in-entire-dom